spec/queries/mentoring_query_spec.rb

```diff
@@ -29,23 +29,34 @@
 
     context 'when there are multiple mentees and mentors' do
       let(:bob) { create(:user) }
-      let!(:bob_mentee) { create(:user, reviewer_id: bob.id) }
-      let!(:other_bob_mentee) { create(:user, reviewer_id: bob.id) }
-
       let(:brown) { create(:user) }
```

Comment:
I'd suggest we switch these names to something more self-explanatory (`:mentor_1`, `:mentor_2`). It will make easier to understand what we want from the expectations.